ibm_catalog.json

```diff
@@ -21,29 +21,30 @@
         "ai"
       ],
       "short_description": "Creates and configures an instance of IBM watsonx.data.",
-      "long_description": "IBM watsonx.data is a data management solution for collecting, storing, querying, and analyzing all your enterprise data with a single unified data platform. The Cloud automation for watsonx.data allows user to set up an instance of watsonx.data. For more details on watsonx.data, [refer here](https://cloud.ibm.com/docs/watsonxdata/index.html).",
+      "long_description": "IBM watsonx.data is a data management solution for collecting, storing, querying, and analyzing all your enterprise data with a single unified data platform. The Cloud automation for watsonx.data allows user to set up an instance of watsonx.data. For more details on watsonx.data, [Learn more](https://cloud.ibm.com/docs/watsonxdata/index.html).",
       "offering_docs_url": "https://github.com/terraform-ibm-modules/terraform-ibm-watsonx-data/blob/main/README.md",
       "offering_icon_url": "https://raw.githubusercontent.com/terraform-ibm-modules/terraform-ibm-watsonx-data/main/images/watsonx-data.svg",
       "provider_name": "IBM",
       "features": [
         {
           "title": "watsonx.data instance",
-          "description": "Get started with provisioning an instance of IBM watsonx.data. Explore more about IBM watsonx.data [here](https://www.ibm.com/products/watsonx-data)."
+          "description": "Get started with provisioning an instance of IBM watsonx.data. Explore more about IBM watsonx.data, [Learn more](https://www.ibm.com/products/watsonx-data)."
```

Comment:
Rephrase the sentence, Learn more does not fit.
After `,`, Learn should not start with `L`(capitol L).